### DISCUSSION TITLE
Documentation feedback

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#21] [Guild Feedback]
Feedback related to the Meta documentation: https://zosopentools.github.io/meta/#/

**Comment by IgorTodorovskiIBM** [comment]
Ideas:
* Goals-based front-page that describes the goals
* z/OS naming inconsistencies
* Improve Latest ports presentation

**Reply by MikeFultonDev** [reply]
Curious what others think about 'goals based front page'. To me, that would be something like particular use cases that people might have and how to go about it, e.g. 'i want to set up a new repo to port software package X', or 'how do i download a software package to use', or 'I found a problem in software package X - how should I report it and/or fix it?'
